search-service/src/main/resources/application.properties

```diff
@@ -29,6 +29,10 @@ management.endpoint.health.enabled = true
 
 spring.mvc.log-request-details = true
 
+# Pagination config for query resources endpoints
+application.pagination.limit-default = 30
+application.pagination.limit-max = 100
+
```

Comment:
You should not have these changes in this PR

Reply:
Strange I really do not know why this is showing in this PR. I didn't even touch search service.